english/bear/bear.md

```diff
@@ -2,4 +2,5 @@ Stella atroir.
 Taiwan bear.
 Ice cold. If warm, the results could be unbearable.
 Bear in mind, the grizzly taste.
-You are now drunk. You stumble over to the nearest cutie and ask, "Hey baby, what's ursine?"
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.